.gitlab/validate-logs-intgs/validate_log_intgs.py

```diff
@@ -79,8 +79,11 @@ def __init__(self, dir_name: str) -> None:
             self.log_collection: bool = 'Category::Log Collection' in content.get('tile', {}).get('classifier_tags', [])
             # boolean: whether or not the integration has public facing docs
             self.is_public: bool = content['display_on_public_website']
-            # Log source defined in the manifest.json of the integration
-            self.log_source: Optional[str] = content.get("assets", {}).get("logs", {}).get("source")
+
+        with open(os.path.join(INTEGRATIONS_CORE, dir_name, f'assets/logs/{dir_name}.yaml'), 'r') as logs_file:
+            content = yaml.load(logs_file)
```

Comment:
nit - I'd use a different name, `content` is already the content of `manifest.json`